functionalTest/api/foo/fooImposterTest.js

```diff
@@ -1,78 +1,62 @@
 'use strict';
 
-var assert = require('assert'),
+const assert = require('assert'),
     api = require('../api').create(),
     tcp = require('../tcp/tcpClient'),
     promiseIt = require('../../testHelpers').promiseIt,
     port = api.port + 1,
-    timeout = parseInt(process.env.MB_SLOW_TEST_TIMEOUT || 2000);
+    timeout = parseInt(process.env.MB_SLOW_TEST_TIMEOUT || 2000),
+    requestName = 'some request name';
```

Comment:
This loses the original context (the "this.name" was passed so you could see the test name in the logs). It's not really important to keep it; probably the best thing is just to drop the name field in all the requests